### PR TITLE
Add Filter And Alarm Not Exists For CMKS query

### LINKED ISSUE #1195: Add Filter And Alarm Not Exists For Customer Master Key Events query for CloudFormation
### Platform
CloudFormation

### Provider
AWS

### Description
A log metric filter and alarm should exist dor Customer Master Key Events

### PR DESCRIPTION
**Pull request by fabioGoncalvesCx**
Added new query for AWS CloudFormation.

Provide a CloudFormation resource and a resource to ensure that  Customer Master Key Events are catched by a log metric filter and an alarm

Closes #1195

### CODE REVIEW

File: assets/queries/cloudFormation/filter_and_alarm_exist_for_customer_master_key_deletion_events/test/positive.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        attaching/detaching policies from IAM users, roles or groups.
+      MetricName: IAMPolicyEventCount
+      Namespace: CloudTrailMetrics
+      Statistic: Sum
+      Period: 300
+      EvaluationPeriods: 1
+      Threshold: 1
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      AlarmActions:
+        - Ref: SnsTopicFake
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/filter_and_alarm_exist_for_customer_master_key_deletion_events/test/positive.yaml:98]: Either remove the reference (intrinsic function `Ref`) to Resource that doesn't exists or add it as a [Parameter](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html) to the header of the template

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
Requesting changes

**Comment by markmishaevcx** [comment]
Scan submitted to Checkmarx

**Comment by markmishaevcx** [comment]
Logo<br />**Checkmarx SAST - Scan Summary & Details**
### Cx-SAST Summary
Total of 4 vulnerabilities<br />High&nbsp;**0 High**<br />Medium&nbsp;**3 Medium**<br />Low&nbsp;**1 Low**<br />Info&nbsp;**0 Info**<br />
### Violation Summary
Medium&nbsp;**3** **Medium**<br />
[View more details on Checkmarx UI](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049056&projectid=727)
### Cx-SAST Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[30](https://github.com/Checkmarx/kics/blob/feature/filter_and_alarm_exist_for_customer_master_key_deletion_events/cmd/builder/main.go#L30) |Medium|Path_Traversal|cmd/builder/main.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049056&projectid=727&pathid=3)
[80](https://github.com/Checkmarx/kics/blob/feature/filter_and_alarm_exist_for_customer_master_key_deletion_events/test/main_test.go#L80) |Medium|Denial_Of_Service_Resource_Exhaustion|test/main_test.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049056&projectid=727&pathid=2)
[52](https://github.com/Checkmarx/kics/blob/feature/filter_and_alarm_exist_for_customer_master_key_deletion_events/pkg/kics/service.go#L52) |Medium|Denial_Of_Service_Resource_Exhaustion|pkg/kics/service.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049056&projectid=727&pathid=1)

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**